### ISSUE TITLE
Installation Issue - LookupRollupSummariesReadOnly: The user license doesn't allow the permission: Read All dlrs__LookupRollupCalculateJob__c

### ISSUE THREAD

**Issue by daveerickson** [afawcett/declarative-lookup-rollup-summaries#331]
I am getting this error while attempting to install to a Partial Copy sandbox:

Package: Declarative Lookup Rollup Summaries Tool (04tb0000000cBqJ)
Error Number: 1584850898-63723 (-943440696)
Problem:

1. (LookupRollupSummariesReadOnly) The user license doesn't allow the permission: Read All dlrs__LookupRollupCalculateJob__c
LookupRollupSummariesReadOnly: The user license doesn't allow the permission: Read All dlrs__LookupRollupCalculateJob__c

I'm not sure where to start troubleshooting this. I appreciate any help.

**Comment by afawcett** [comment]
What license type does your user have? I did some research and found [this](https://developer.salesforce.com/forums/?id=906F00000009BSSIA2) post, which suggests it could be a restriction of your user license.

**Comment by daveerickson** [comment]
I have a Salesforce user license. The real oddity is I can install successfully in a dev sandbox, but not a partial copy or full copy sandbox.

I should have probably pointed out that this is an upgrade, not a clean install. Not sure what difference that could make.

I can grab some logs if that would help, or grant login access. Thanks for your help.

**Comment by afawcett** [comment]
Wow that is really odd, i can only suggest at this point to raise it with Salesforce support to get an explanation, sorry i cannot be of further assistance, if i think of something i'll for sure let you know. Sorry.

**Comment by daveerickson** [comment]
I opened a case with premier support and have have some details on what is causing the issue.

I have assigned the permission set LookupRollupSummariesReadOnly to all users in my org, including partner community login users. That license type does not allow the View All permission to custom objects. The permission set is adding View All to dlrs__LookupRollupCalculateJob__c.

The install succeeded in my dev org as there are no community users records, but there are in my partial copy, full, and production. The suggestion from premier support is to remove the permission set from all partner users, then try the upgrade again. Even if that works I don't think that is what I want to do. If they had access to change a value on a record I'd want their change to also trigger the recalculation of the parent.

Looking at the permission set this object is the only one that has View All set to true. Is that required for DLRS to function? I checked the org wide defaults for that object in my dev org at it is Public Read/Write same as Lookup Rollup Summary is, so I'm not sure if the view all is strictly needed. I'd be happy to try out an install in my partial sandbox with the View All permission removed and see if it works.

Thanks again for your support, and for an excellent tool.
